python/ray/tests/utils.py

```diff
@@ -94,3 +94,57 @@ def wait_for_errors(error_type, num_errors, timeout=10):
             return
         time.sleep(0.1)
     raise Exception("Timing out of wait.")
+
+
+def wait_for_success_output(process_handle, timeout=10):
```

Comment:
This is only used in the multi_node_test, right? Let's move it back into that file.